test/functional/bundleadd/add-install-time.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n bundle1 -f /file1,/file2 "$TEST_NAME"
+
+}
+
+test_teardown() {
```

Comment:
It is ok to leave it if you want but FYI, If you don't specify a test_teardown, the default test_teardown will be used which has _destroy_test_environment "$TEST_NAME"_  so you could remove the test_teardown in this case.